### PR TITLE
utilize vmap attributes

### PR DESCRIPTION
**Pull request by thanrobinson**

### CODE REVIEW

**Comment by thanrobinson** [comment]
[hii_rail temporal automation](https://trello.com/c/Cq5svA9s/207-hiirail-temporal-automation)

File: src/hii_rail.py [hunk 1]
```diff
@@ -127,29 +142,49 @@ def osm_vmap_combined_influence(self):
...
+        vmap_indirect_weights = vmap_direct_weights.multiply(
+            self.DIRECT_INDIRECT_INFLUENCE_RATIO
+        ).select(band_names.get("vmap"))
+
+        rail_direct_cost = (
+            self.vmap.select(band_names.get("vmap"))
+            .distance(kernel=self.kernel["DIRECT"], skipMasked=False)
+            .lte((self.DIRECT_INFLUENCE_WIDTH - self.NOMINAL_RAIL_WIDTH) / 2)
+            .multiply(vmap_direct_weights)
+            # .reduce(ee.Reducer.max())
```
Comment (saanobhaai) [inline src/hii_rail.py:167]: Why are these lines commented? Don't we need these?

**Review by saanobhaai — CHANGES_REQUESTED** [review]
Try as I might, I could find only one thing to flag...

File: src/hii_rail.py (hunk 1 above)
Comment (thanrobinson) [reply src/hii_rail.py:167]: these can actually be deleted. I had to move the max reducer to a subsequent step.

**Pull request merged**